### DISCUSSION TITLE
Plotting getting slower-Phase 1 taking more time in staggered plots

### DISCUSSION THREAD

**Discussion by faroukhnaveed** [swar/Swar-Chia-Plot-Manager#706] [General]
Hi Guys,
I have observed that parallel plotting keeps getting slower , what I mean to say is that the phase 1 time for the plots increases with every consecutive plot on the same drive. It could be my machine or how I am setting the plots up but I wanted to know if others have experienced the same behaviour ? I have pasted the output of my plotting below as you can see the phase 1 plot time is increasing for the same drive, I am using 2 drives and both of them are showing the same behaviour. I am staggering them and adding a delay of around 75 to 85 mins.

1     mp510-n-drive   32   37da830   8016    2021-05-21 10:08:56     07:17:22       3       **02:46** / 02:40   67.19%     192 GiB

3     mp510-n-drive   32   bef11e9   2392    2021-05-21 11:33:59      05:52:19       2       **03:56**               49.50%     224 GiB

5     mp510-n-drive   32   520d5da   2976    2021-05-21 12:59:01     04:27:17       2       **04:22**               37.73%     158 GiB

I am running a 1st gen 1900X threadripper (I know its old :-)), 32gb RAM.
Thanks.

**Comment by Meyer85** [comment]
What I found is that a secondary temp drive speeds this up dramatically.

Buy a 1TB 2.5" sata SSD WD green or something. Use this drive as temp to for plot staging (phase 3 and 4 happens here) and the final write is from this drive to your spinning drives. This will free up your NVME SSD's to keep plotting full speed on phase 1 and 2

**Reply by Ron-ski** [reply]
How much space does phase 3 & 4 require per plot?

**Comment by Ron-ski** [comment]
I observed this with two Firecuda 520 Drives. These and others have a high speed cache, once this fills the drive slows down drastically.

Let's say that once the cache is full it writes at 500MB/s, now running two plots in phase 1 at the same time means each plot is writing at 250MB/s, four plots at the same to time would be 125MB/s each.

So as your staggered plots kick in others are still running in phase 1, and thus everything slows down.
